### PR TITLE
Readme edit

### PR DESCRIPTION
**Pull request by code-jace**
* Added Seneca logo to header,
* Updated badges,
* Updated contact details,
* Updated *Contributing* with link,
* Fixed function syntax.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,15 +38,16 @@ npm run test
...
-  host: '127.0.0.1',
-  port: 27017
+seneca.use('mongo-store',{
+  name:'dbname',
+  host:'127.0.0.1',
+  port:27017
 })
 
-seneca.ready(function(){
+seneca.ready(){
```
Comment (mcdonnelldean) [inline README.md]: Add function back here it should read `seneca.ready(function () {`

File: README.md [hunk 2]
```diff
@@ -46,21 +58,23 @@ seneca.ready(function(){
...
 
-entity.save$(function (err, entity) { ... })
-entity.load$({id: ...}, function (err, entity) { ... })
-entity.list$({property: ...}, function (err, entity) { ... })
-entity.remove$({id: ...}, function (err, entity) { ... })
+entity.save$( function (err,entity) { ... } )
+entity.load$( {id: ...}, function (err,entity) { ... } )
+entity.list$( {property: ...}, function (err,entity) { ... } )
+entity.remove$( {id: ...}, function (err,entity) { ... } )
 ```
```
Comment (mcdonnelldean) [inline README.md]: remove padding on each end and add a space after commas eg `entity.save$(function (err, entity) { ... })`

File: README.md [hunk 3]
```diff
@@ -105,24 +120,32 @@ orders_ent.native$(function (err, db) {
...
-You can also use: `entity.list$({f1:v1,...}, {native$:[{-mongo-query-}, {-mongo-options-}]})` which
-allows you to specify a native mongo query per [node-mongodb-native][]
+You can also use: `entity.list$({f1:v1,...}, {native$:[{-mongo-query-}, {-mongo-options-}]})` which allows you to specify
+a native mongo query per [node-mongodb-native][]
 
 ## Contributing
-We encourage participation. If you feel you can help in any way, be it with
-examples, extra testing, or new features please get in touch.
+The [Senecajs org][] encourage open participation. If you feel you can help in any way, be it with
+documentation, examples, extra testing, or new features please get in touch.
```
Comment (mcdonnelldean) [inline README.md]: encourages

File: README.md (hunk 1 above)
Comment (code-jace) [inline README.md]: Understood

**Comment by mcdonnelldean** [comment]
@code-jace Good job!

**Pull request merged**